_posts/_2021-10-05-Launch-Announcement-1-1.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+authors: 
+  - henkle
+  - ahopp
+comments: true
+title: "OpenSearch 1.1 is here!"
+category:
+- releases
+---
+
+To the OpenSearch community everywhere, 
+
+We are excited to announce the 1.1 release of the OpenSearch project (available to [download](https://opensearch.org/downloads.html) today). Not only are we excited that this release marks the first minor version release on the OpenSearch project, it also comes with a number of new and exciting features. And, best of all, we continue to see the community grow and contribute to OpenSearch!
+
+So, what’s in 1.1? This release includes dozens of new features and hundreds of improvements (Release Notes) but we wanted to highlight some of the most exciting new features in this release;
```

Comment:
needs link for release notes. https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-1.1.0.md as per https://github.com/opensearch-project/opensearch-build/pull/478/files

Reply:
I think it should end with a `:` not a `;`